bedrock/firefox/templates/firefox/new/onboarding/scene1.html

```diff
@@ -117,10 +131,20 @@ <h2 class="forward">{{_('Go Forward') }}</h2>
 
   <div id="other-platforms">
     <div class="container">
-      <section class="section-other-platforms">
-        <h4>{{_('Firefox for other platforms') }}</h4>
+      {#
+        when l10n tag 'firefox_new_install_options' is removed, remove
+        'no-lang-link' class below and from the CSS file
+      #}
+      <section class="section-other-platforms{% if l10n_has_tag('firefox_new_install_options') %} no-lang-link{% endif %}">
+        <h4>
+        {% if l10n_has_tag('firefox_new_install_options') %}
+          {{ _('Advanced Install Options & Other Platforms') }}
```

Comment:
Correction: we might have a problem. 

The same string is used both for the link and for the title. Sadly that's a known limitation of lang file (string is the ID).

Any chance that we can differentiate them? 

P.S. is it just me or title case is normally not used in links?